### ISSUE TITLE
Main search input is not labelled

### ISSUE THREAD

**Issue by LukasHirt** [owncloud/web#1649]

**Comment by marcus-herrmann** [comment]
# Problem (abstract)
For a user of assistive technologies such as screenreaders it is not clear what this input is actually for, especially since no button exists

# Solution (abstract)
[Visually hidden](https://a11yproject.com/posts/how-to-hide-content/) label, or the use of aria-label, that you are using already. 

# Solution in phoenix
(wip)

**Comment by PVince81** [comment]
I'll put this to milestone 1 because this empty field gives a bad first impression

**Comment by LukasHirt** [comment]
Search now has both placeholder and aria-label with "Search" value
